src/views/Main.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+  <div class="main">
+    <Header />
+  </div>
+</template>
+
+<script>
+import Header from '../components/Header.component.vue'
+
+export default {
+  name: 'Main',
+  components: {
+    Header,
```

Comment:
이거는 취향차이이지만, 저희는 케밥케이스 컴포넌트를 사용하고 있어서 코드 스타일 맞추면 좋을 것 같아요
```suggestion
    'header': Header,
```